cmd/jaeger/internal/exporters/storageexporter/config.go

```diff
@@ -19,6 +20,7 @@ var (
 type Config struct {
 	TraceStorage string                          `mapstructure:"trace_storage" valid:"required"`
 	QueueConfig  exporterhelper.QueueBatchConfig `mapstructure:"queue" valid:"optional"`
+	RetryConfig  configretry.BackOffConfig       `mapstructure:"retry_on_failure" valid:"optional"`
```

Comment:
I am not sure what `valid:"optional"` means in the context of having an actual struct by value.